backend/msgpack/queue.go

```diff
@@ -160,16 +160,14 @@ type queueMetrics struct {
 	connWriteErrors   tally.Counter
 }
 
-func newQueueMetrics(s tally.Scope, instanceID string, queueSize int) queueMetrics {
+func newQueueMetrics(s tally.Scope, queueSize int) queueMetrics {
 	numBuckets := defaultQueueSizeNumBuckets
 	if queueSize < numBuckets {
 		numBuckets = queueSize
 	}
 	buckets := tally.MustMakeLinearValueBuckets(0, float64(queueSize/numBuckets), numBuckets)
 	return queueMetrics{
-		queueSize: s.Tagged(
-			map[string]string{"instance-id": instanceID},
-		).Histogram("queue-size", buckets),
+		queueSize: s.Histogram("queue-size", buckets),
```

Comment:
Yea, though we could use `excludeByTags` to remove them. Either way, I like the name `queue-length` better since size can be a little ambiguous (is it the capacity of the length of the queue). Will update.